localstack-core/localstack/openapi.yaml

```diff
@@ -516,6 +544,8 @@ paths:
   /_localstack/config:
     get:
       description: Get current LocalStack configuration
+      operationId: get_localstack_config
+      tags: [internal]
```

Comment:
have you considered something like `localstack` instead of `internal`? The latter might suggest that it's not meant for public use.

furthermore, if the tag is set to `localstack`, you could set operation ID to `get_config`

Reply:
Good input, `localstack` fits much better (it is actually also what we used in our docs).